lib/usd/translators/CMakeLists.txt

```diff
@@ -82,6 +82,7 @@ target_link_libraries(${TARGET_NAME}
         usdUtils
         vt
         ${MAYA_LIBRARIES}
+        OpenMayaFX
```

Comment:
I know @robthebloke has added this before and it's been rejected (I can't find that PR at the moment) but it's definitely needed as a dependency at least in maya-2019, when running our tests using ALs test/build infrastructure

Reply:
For what it's worth, we definitely needed this when usdMaya was in the USD repo (v19.11):
https://github.com/PixarAnimationStudios/USD/blob/4b1162957b2ad219e1635c81de8343be490e41fc/third_party/maya/plugin/pxrUsdTranslators/CMakeLists.txt#L24

This was because the translator for Maya `stroke` nodes used an OpenMayaFX header:
https://github.com/PixarAnimationStudios/USD/blob/4b1162957b2ad219e1635c81de8343be490e41fc/third_party/maya/plugin/pxrUsdTranslators/strokeWriter.cpp#L44

Reply:
Thanks @mattyjams ... The link error we're seeing is related to [MFnParticleSystem](https://help.autodesk.com/view/MAYAUL/2019/ENU/?guid=Maya_SDK_MERGED_cpp_ref_class_m_fn_particle_system_html) in [particleWriter.cpp](https://github.com/Autodesk/maya-usd/blob/1c48ce453961448b2916248322449b3737437f87/lib/usd/translators/particleWriter.cpp#L36)